src/core/lib/gprpp/dual_ref_counted.h

```diff
@@ -68,15 +69,17 @@ class DualRefCounted : public Orphanable {
       std::enable_if_t<std::is_base_of<Child, Subclass>::value, bool> = true>
   RefCountedPtr<Subclass> RefAsSubclass() {
```

Comment:
Please also change the `WeakRefAsSubclass()` methods.

Reply:
done